docs/server-setup.md

```diff
@@ -66,10 +66,11 @@ To run your tests against a remote Selenium Server, you will need an account wit
 In your config file, set these options:
  - `sauceUser` -  The username for your Sauce Labs account.
  - `sauceKey` -  The key for your Sauce Labs account.
-
+ - `sauceSeleniumAddress` - default: ondemand.saucelabs.com:80/wd/hub ; travis: localhost:4445/wd/hub
```

Comment:
nit: sauceSeleniumAddress is not a mandatory field for users to set. I think it's good to mention this, but it should sound more like a tip for a user.